pkgs/applications/misc/dnd-tools/default.nix

```diff
@@ -0,0 +1,29 @@
+{ python3, fetchFromGitHub, fetchpatch, lib }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dnd-tools";
+  version = "baefb9e";
```

Comment:
```suggestion
  version = "unstable-2021-02-19";
```

Reply:
I take it you meant `unstable-2021-02-18` right?

Reply:
18 will be better. GitHub shows as 19 Feb but actual commit date is 18th.

Reply:
GitHub shows 18th for me, maybe a bug with timezones or something idk.